test/hotspot/jtreg/ProblemList.txt

```diff
@@ -198,7 +198,11 @@ vmTestbase/nsk/jvmti/ResourceExhausted/resexhausted001/TestDescription.java 7013
 vmTestbase/nsk/jvmti/ResourceExhausted/resexhausted003/TestDescription.java 6606767 generic-all
 vmTestbase/nsk/jvmti/ResourceExhausted/resexhausted004/TestDescription.java 7013634,6606767 generic-all
 vmTestbase/nsk/jvmti/ThreadStart/threadstart001/TestDescription.java 8016181 generic-all
+<<<<<<< HEAD
 vmTestbase/nsk/jvmti/ThreadStart/threadstart003/TestDescription.java 8034084 generic-all
+=======
+vmTestbase/nsk/jvmti/scenarios/extension/EX03/ex03t001/TestDescription.java 8173658 generic-all
+>>>>>>> 14bb47410b... 8034084: nsk.nsk/jvmti/ThreadStart/threadstart003 Wrong number of thread end events
```

Comment:
That stuff does not look correctly resolved ;-)

Reply:
you have a point there ... :)